frontend/src/pages/AccountSend.vue

```diff
@@ -397,6 +399,18 @@ function useSendForm() {
 
     // Validate the form
     await sendFormRef.value?.validate();
+
+    // Switch chain
+    const chain = supportedChains.filter((chain) => chain.chainId === `0x${Number(linkChainId)?.toString(16)}`);
+    if (
+      chain.length === 1 &&
+      !isLoading.value &&
+      chainId.value !== Number(linkChainId) &&
+      !attemptedNetworkChange.value
+    ) {
+      attemptedNetworkChange.value = true;
+      await setNetwork(chain[0]);
```

Comment:
Should we reset `attemptedNetworkChange` to false after the `await setNetwork` line? Also why exactly do we need the `attemptedNetworkChange` var?

Reply:
I see, thanks!